redisinsight/api/src/modules/rdi/entities/rdi.entity.ts

```diff
@@ -7,32 +7,20 @@ export class RdiEntity {
   @PrimaryGeneratedColumn('uuid')
   id: string;
 
-  @Expose()
-  @Column({ nullable: false })
-  type: string;
-
   @Expose()
   @Column({ nullable: true })
   url: string;
 
-  @Expose()
-  @Column({ nullable: true })
-  host?: string;
-
-  @Expose()
-  @Column({ nullable: true })
-  port?: number;
-
   @Expose()
   @Column({ nullable: false })
   name: string;
 
   @Expose()
-  @Column({ nullable: true })
+  @Column({ nullable: false })
   username: string;
```

Comment:
Are you sure that username and password are required fields?

Reply:
Yep. We have the same logic on frontend side. And RDI team confirmed that username and password are mandatory fields. Username can be blank string.